commands/sql/sql.drush.inc

```diff
@@ -139,7 +141,7 @@ function sql_drush_command() {
       ),
       'file' => 'Path to a file containing the SQL to be run. Gzip files are accepted.',
       'extra' => array(
-        'description' => 'Add custom options to the mysql command.',
+        'description' => 'Add custom options to the SQL command.',
```

Comment:
Maybe "Add extra arguments to the database connection. Perhaps add an item to the Examples section of the command definition that illustrates.